docs/node: Improve introduction on Validator node (#401)
oasisprotocol/docs: docs/node/run-your-node/validator-node/README.md

# Validator Node

This guide will walk you through the process of setting up your **validator node** 
for the Oasis Network on either the Mainnet or the Testnet. It is designed for 
individuals who have a basic understanding of command line tools. 

## Prerequisites

Before proceeding with this guide, ensure that you have completed the steps 
outlined in the [Prerequisites] section so that:

> * Your system meets the [hardware requirements]

Reviewer: Let's use bullets as they were all the same sentence. So:

```suggestion
* your system meets the [hardware requirements],
```

> * You have the [Oasis Node binary] installed on your systems.

Reviewer: ```suggestion
* you have the [Oasis Node binary] installed on your systems,
```

> * You understand what are [Stake requirements] to become a validator on the

Reviewer: ```suggestion
* you understand what are [Stake requirements] to become a validator on the
```

>  Oasis Network

Reviewer: ```suggestion
 Oasis Network.
```

[Prerequisites]: ../prerequisites/
[hardware requirements]: ../prerequisites/hardware-recommendations.md
[Oasis Node binary]: ../prerequisites/oasis-node.md
[Stake requirements]: ../prerequisites/stake-requirements.md

## Physical Machines

You will be using two separate physical machines for deployment: your local

>  system, referred to as `localhost`, and a remote `server` which will function

Reviewer: The space in the beginning of lines is redundant.

 as an Oasis node. The reason for this is to ensure protection of the keys used
 to setup your node.

**Localhost:** On the localhost, you will [create your entity] which is

>  essential for deploying nodes on the network. To ensure the security of these

Reviewer: ```suggestion
 essential for deploying nodes on the network. To ensure the security of your
```

>  items, we strongly recommend to isolate the localhost from any network or

Reviewer: ```suggestion
private keys, we strongly recommend to isolate the localhost from any network or
```

 internet connectivity. Once the keys have been generated, it is highly

>  recommended to use Hardware Security Modules (HSM) for secure storage. 

Reviewer: ```suggestion
recommended to use Hardware Security Modules (HSM) for secure storage
such as the [Ledger wallet].
 
 [Ledger wallet]: ../../../general/manage-tokens/holding-rose-tokens/ledger-wallet
```

## Prerequisites

Before proceeding with this guide, ensure that you have completed the steps 
outlined in the [Prerequisites] section so that:
* your system meets the [hardware requirements],
* you have the [Oasis Node binary] installed on your systems,
* you understand what are [Stake requirements] to become a validator on the
 Oasis Network.

[Prerequisites]: ../prerequisites/
[hardware requirements]: ../prerequisites/hardware-recommendations.md
[Oasis Node binary]: ../prerequisites/oasis-node.md
[Stake requirements]: ../prerequisites/stake-requirements.md

## Physical Machines

You will be using two separate physical machines for deployment: your local
system, referred to as `localhost`, and a remote `server` which will function
 as an Oasis node. The reason for this is to ensure protection of the keys used
 to setup your node.

**Localhost:** On the localhost, you will [create your entity] which is
essential for deploying nodes on the network. To ensure the security of these
private keys, we strongly recommend to isolate the localhost from any network or
internet connectivity. Once the keys have been generated, it is highly

> recommended to use Hardware Security Modules (HSM) for secure storage

Reviewer: I would also add a link
```suggestion
recommended to use [Hardware Security Modules (HSM)] for secure storage
```

to https://en.wikipedia.org/wiki/Hardware_security_module

such as the [Ledger wallet].

[Ledger wallet]:
  ../../../general/manage-tokens/holding-rose-tokens/ledger-wallet

**Server:** After the entity has been created, proceed to [set up the server in
 order to start the Oasis node]. The server needs to meet the hardware 
 requirements and have an acces to the internet.

> [create your entity]: ./#creating-your-entity

Reviewer: I think `./` is redundant in this case. See https://docusaurus.io/docs/markdown-features/toc#heading-ids
```suggestion
[create your entity]: #creating-your-entity
```
